coffee_lib/src/plugin.rs

```diff
@@ -26,26 +26,31 @@ pub enum PluginLang {
 }
 
 impl PluginLang {
-    pub async fn default_install(
+    pub async fn get_executable_path(
```

Comment:
```suggestion
    pub async fn get_executable_path(
```

Why? this function perform the default installation of the plugin